AppCreationScripts/Apps.json

```diff
@@ -36,7 +36,7 @@
             {
               "id": "b340eb25-3456-403f-be2f-af7a0d370277",
               "type": "Scope",
-              "x-ms-name": "User.ReadBasic.All"
```

Comment:
Does ReadBasic.All need admin permission? The portal does not list it as a scope that need admin consent
![UserReadBasic](https://user-images.githubusercontent.com/35743865/66941554-fbb0ab80-effb-11e9-9e97-cb3052fbe0ee.PNG)

Reply:
Sorry I think my comment was wrong. My intention is that the step adding permissions can be avoided totally as user.read is automatically included when the app is registered.

Reply:
I like the idea of simplifying to use the /me call in MS Graph but we will need to update the quickstart docs here as well: 
https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-v2-java-webapp

Reply:
Sure will update in Quickstart too.